core/cautils/getter/kscloudapi.go

```diff
@@ -136,17 +133,38 @@ func (api *KSCloudAPI) Get(fullURL string, headers map[string]string) (string, e
 	return HttpGetter(api.httpClient, fullURL, headers)
 }
 
-func (api *KSCloudAPI) GetAccountID() string          { return api.accountID }
-func (api *KSCloudAPI) IsLoggedIn() bool              { return api.loggedIn }
-func (api *KSCloudAPI) GetClientID() string           { return api.clientID }
-func (api *KSCloudAPI) GetSecretKey() string          { return api.secretKey }
-func (api *KSCloudAPI) GetFrontendURL() string        { return api.feURL }
-func (api *KSCloudAPI) GetApiURL() string             { return api.apiURL }
-func (api *KSCloudAPI) GetAuthURL() string            { return api.authURL }
-func (api *KSCloudAPI) GetReportReceiverURL() string  { return api.erURL }
+func (api *KSCloudAPI) GetAccountID() string   { return api.accountID }
+func (api *KSCloudAPI) IsLoggedIn() bool       { return api.loggedIn }
+func (api *KSCloudAPI) GetClientID() string    { return api.clientID }
+func (api *KSCloudAPI) GetSecretKey() string   { return api.secretKey }
+func (api *KSCloudAPI) GetCloudReport() string { return api.cloudReport }
+func (api *KSCloudAPI) GetCloudAPI() string    { return api.cloudAPI }
+func (api *KSCloudAPI) GetCloudUI() string     { return api.cloudUI }
+func (api *KSCloudAPI) GetCloudAuth() string   { return api.cloudAuth }
+
 func (api *KSCloudAPI) SetAccountID(accountID string) { api.accountID = accountID }
 func (api *KSCloudAPI) SetClientID(clientID string)   { api.clientID = clientID }
 func (api *KSCloudAPI) SetSecretKey(secretKey string) { api.secretKey = secretKey }
+func (api *KSCloudAPI) SetCloudReport(cloudReport string) {
+	if cloudReport != "" {
```

Comment:
I'm not sure we want logic in the setters except overriding... (what if an empty value was intentional?)